README.md

```diff
@@ -138,19 +136,26 @@ The virtual staining method is described in [this preprint](https://doi.org/10.1
 }
 ```
 
-We also report the application of the virtual staining pipeline in [this preprint](https://doi.org/10.1101/2023.12.19.572435):
+The previous version of virtual staining was reported in [this paper](https://elifesciences.org/articles/55502):
```

Comment:
I think we can just list the citations and sort them with most recent first.

Reply:
Not strong opinion, but we will start having more CV applications in this repo

Reply:
I would a) shorten the reference and b) move it up to the section before `Installing VisCy`

Reply:
Implementing these changes.